caffe-grid/pom.xml

```diff
@@ -18,11 +18,28 @@
     <maven.compiler.target>1.7</maven.compiler.target>
     <hadoop.version>2.5.0</hadoop.version>
     <scala.version>2.10.4</scala.version>
```

Comment:
Looking more closely - you have it backwards: we need the user to specify the 
    
   -Dscala.version=2.11.7

on the command line.  We should not hard-code the ".7".

Instead we need logically:

<scala.major.version>substr(${scala.version), lastIndexOf(${scala.version},',')</scala.major.version>

Obviously maven does not support that. What I will use most likely:  "maven build-helper plugin" which has a regex target.  Will update in few minutes.